pkg/exporter/process.go

```diff
@@ -305,6 +316,32 @@ func (ep *ExportingProcess) createAndSendMsg(set entities.Set) (int, error) {
 	return bytesSent, nil
 }
 
+// createAndSendJSONMsg takes in a set as input, creates the JSON record, and sends it out.
+func (ep *ExportingProcess) createAndSendJSONMsg(set entities.Set) (int, error) {
+	var bytesSent int
+	for _, record := range set.GetRecords() {
+		records := make(map[string]interface{})
```

Comment:
nit: is "records" an accurate variable name? It seems to be a collection of fields from records.

Reply:
Updated. Thanks